mpas_analysis/test/test_climatology.py

```diff
@@ -296,6 +296,9 @@ def test_cache_climatologies(self):
                                                             printProgress=True)
             self.assertArrayApproxEqual(dsClimatology.mld.values,
                                         refClimatology.mld.values)
+
+            self.assertEqual(dsClimatology.attrs['totalMonths'], 3)
+            self.assertApproxEqual(dsClimatology.attrs['totalDays'], 89.958333)
```

Comment:
Why `89.958333`?  This isn't obvious.  A comment on this test and the one below would be really helpful.